code/analytics/src/test/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsHitProcessorTests.kt

```diff
@@ -333,8 +349,21 @@ class AnalyticsHitProcessorTests {
 
     @Test
     fun `network success - assurance enabled`() {
+        // Set up mock Assurance Shared State
+        val mockAssuranceSharedStateData = mapOf("integrationid" to "testIntegrationId")
+        val mockAssuranceSharedStateResult = SharedStateResult(SharedStateStatus.SET, mockAssuranceSharedStateData)
+        Mockito.`when`(
+            mockedExtensionApi.getSharedState(
+                Mockito.eq("com.adobe.assurance"),
+                Mockito.eq(null),
+                Mockito.eq(false),
+                Mockito.eq(SharedStateResolution.LAST_SET)
+            )
+        ).thenReturn(mockAssuranceSharedStateResult)
         Mockito.`when`(mockedAnalyticsState.isAssuranceSessionActive).thenReturn(true)
+        updateMockedSharedState("com.adobe.assurance", mapOf("integrationid" to "testIntegrationId"))
```

Comment:
I don't think this is needed. Remove the updateSharedstate method and mockedSharedState variable.